src/cards/ha-camera-card.html

```diff
@@ -49,76 +51,78 @@
 </dom-module>
 
 <script>
-Polymer({
-  is: 'ha-camera-card',
-  UPDATE_INTERVAL: 10000, // ms
-
-  properties: {
-    hass: {
-      type: Object,
-    },
-
-    stateObj: {
-      type: Object,
-      observer: 'updateCameraFeedSrc',
-    },
-
-    cameraFeedSrc: {
-      type: String,
-    },
-
-    imageLoaded: {
-      type: Boolean,
-      value: true,
-    },
-
-    /**
-     * The z-depth of the card, from 0-5.
-     */
-    elevation: {
-      type: Number,
-      value: 1,
-      reflectToAttribute: true,
-    },
-  },
-
-  listeners: {
-    tap: 'cardTapped',
-  },
-
-  attached: function () {
-    this.timer = setInterval(
-      function () {
-        this.updateCameraFeedSrc(this.stateObj);
-      }.bind(this),
-      this.UPDATE_INTERVAL
-    );
-  },
-
-  detached: function () {
+const UPDATE_INTERVAL = 10000; // ms
```

Comment:
This is now defined in the global scope of the whole page, that's not good.

We either use the fact that `const` is scoped to the block and wrap our class in

```js
{
  const UPDATE_INTERVAL = 10000; // ms
  class HaCameraCard {
    …
  }
}
```

or we decide to leave it as is because the conversion to Polymer 3 will make it module scoped anyway.